contrib/python-hatch-fancy-pypi-readme/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "python-hatch-fancy-pypi-readme"
```

Comment:
we patched this out everywhere so far, but i guess we could just keep it, since it's a tiny module and saves some patches (with no downside to having it in build hostdeps)

Reply:
i'd prefer patching it out, since it's entirely useless, and it's very little work to patch it out